nucypher/cli/status.py

```diff
@@ -38,8 +38,8 @@ def status(click_config, config_file):
     #
     ursula_config = UrsulaConfiguration.from_configuration_file(filepath=config_file)
     if not ursula_config.federated_only:
-        ursula_config.connect_to_blockchain(provider_uri=ursula_config.provider_uri)
-        ursula_config.connect_to_contracts()
+        ursula_config.get_blockchain_interface(provider_uri=ursula_config.provider_uri)
```

Comment:
`get_blockchain_interface` reads a little weird since nothing is being returned...is this more of a setup/connect type function? I'm unfamiliar with other uses of the same function but maybe a rename?

Reply:
Good eye - I have also been thinking this.  I'd like to deprecate this function all together.  What do you think?

Reply:
I could be 😄 ...deprecate it in favour of what?

Reply:
Well - I think the instantiation of `BlockchainInterface` can be done explicitly where needed.

Reply:
I'd be +1 for that if it isn't needed to be instantiated here and can be done later.

Reply:
:+1: - Id like to make this change in a follow-up PR.